src/content/topics/sdk/server-side/python.mdx

```diff
@@ -45,6 +45,23 @@ import ldclient
 </CodeTabItem>
 </CodeTabs>
 
+Additionally, you will need to pass a `Config` object to the `ldclient.set_config` method to instantiate your client instance. You can import the `Config` object with this line:
+
+<CodeTabs
+  defaultValue="py"
+  values={[
+    { label: 'Python', value: 'py', },
+  ]
+}>
+<CodeTabItem value="py">
+
+```py
+from ldclient.config import Config
+```
+
+</CodeTabItem>
+</CodeTabs>
+
```

Comment:
To minimize the number of steps needed to get started, what about adding this additional import to the preceding step? That way we import both `ldclient` and `ldclient.config` in the same step instead of extending the documentation section.